components/webfield/ReviewerConsole.js

```diff
@@ -559,16 +576,16 @@ const ReviewerConsole = ({ appContext }) => {
       />
       <Tabs>
         <TabList>
-          <Tab id="assigned-papers" active>
-            Assigned Papers
+          <Tab id={`assigned-${submissionName}`} active>
```

Comment:
I don't think that ids or other values not displayed to the users should be parameterized. These could all stay as "papers" and not have any change in functionality.